### ISSUE TITLE
HTMLElement.rootElement is deprecated.

### ISSUE THREAD

**Issue by vrutkovs** [dsandstrom/atom-gutter-shadow#12]
Atom 1.19 beta0 suggested to file an issue, so I did

The contents of `atom-text-editor` elements are no longer encapsulated
within a shadow DOM boundary. Please, stop using `rootElement` and access
the editor contents directly instead.
```
HTMLElement.rootElement (/app/share/atom-beta/resources/app/src/text-editor-element.js:23:10)
new GutterScrollView (/home/vrutkovs/.atom/packages/gutter-shadow/lib/gutter-scroll-view.coffee:6:35)
<unknown> (/home/vrutkovs/.atom/packages/gutter-shadow/lib/gutter-shadow.coffee:30:28)
Workspace.observeTextEditors (<embedded>:26080:15)
Object.activate (/home/vrutkovs/.atom/packages/gutter-shadow/lib/gutter-shadow.coffee:26:50)
Package.activateNow (<embedded>:69225:25)
```

**Comment by vrutkovs** [comment]
Also enabling this package breaks editor functions - mouse click doesn't change text cursor position

**Comment by dsandstrom** [comment]
Thanks for the issue, please try v0.8.0.

**Comment by dsandstrom** [comment]
This doesn't fix the mouse problems, that seems to be a different bug.

**Comment by dsandstrom** [comment]
Fixed by v1.0.0
